src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import moment from 'moment';
+import Moment from 'moment';
```

Comment:
moment is a function, the same as classnames is a function, that we import directly from a dependency. It is not a class, like the React class we import from react. Functions start with lowercase letter, Classes start with uppercase.

Reply:
Fixes done, went with Moment (its namespace)

Reply:
This uses camel case: https://www.typescriptlang.org/docs/handbook/namespaces.html